lib/components/timer_component.dart

```diff
@@ -0,0 +1,21 @@
+import 'dart:ui';
+import './component.dart';
+import '../time.dart';
+
+/// Simple component which wraps a [Timer] instance allowing it to be easily used inside a [BaseGame] game.
+class TimerComponent extends Component {
+  Timer timer;
+
+  TimerComponent(this.timer);
```

Comment:
should we start on create? not sure, I don't have a strong opinion about it, just mentioning another possibility to see what you prefer. I'm totally down for both.